lib/Entity/DisplayEvent.php

```diff
@@ -165,8 +165,11 @@ public function eventEndByReference(int $displayId, int $eventTypeId, int $refId
             )
         );
 
+        // When updating the event end, concatenate the end message to the current message
         $this->getStore()->update(
-            'UPDATE `displayevent` SET `end` = :toDt
+            'UPDATE `displayevent` SET 
+                      `end` = :toDt, 
+                      `detail` = CONCAT_WS(". ", NULLIF(CONCAT_WS(".", NULLIF(`detail`, "")), ""), :detail)
```

Comment:
Can we use single quotes for consistency?

Reply:
Also can you explain why the double concat_ws in a comment so its easy to refer to later?